### PR TITLE
Change Courant tests to use relative tolerances

### PR DESCRIPTION
**Pull request by simonbyrne**
Since the result is quite small and the calculation is multiplicative, it makes sense to use a relative tolerance here.

### CODE REVIEW

File: test/DGmethods/courant.jl [hunk 1]
```diff
@@ -131,27 +131,10 @@ let
...
-                    @test abs(courant(nondiffusive_courant, dg, model, Q, Δt, VerticalDirection()) - c_v) <= 1e-16
-                    @test abs(courant(diffusive_courant,    dg, model, Q, Δt, VerticalDirection()) - d_v) <= 1e-8 
-                else
-                    @test abs(courant(nondiffusive_courant, dg, model, Q, Δt, HorizontalDirection()) - c_h) <= 1e-11
-                    @test abs(courant(diffusive_courant,    dg, model, Q, Δt, HorizontalDirection()) - d_h) <= 1e-4
-                    @test abs(courant(nondiffusive_courant, dg, model, Q, Δt, VerticalDirection()) - c_v) <= 1e-11
-                    @test abs(courant(diffusive_courant,    dg, model, Q, Δt, VerticalDirection()) - d_v) <= 2e-8
-                end
+                @test courant(nondiffusive_courant, dg, model, Q, Δt, HorizontalDirection()) ≈ c_h rtol=1e-4
+                @test courant(diffusive_courant,    dg, model, Q, Δt, HorizontalDirection()) ≈ d_h rtol=1e-4
```
Comment (simonbyrne) [inline test/DGmethods/courant.jl]: Either the calculation or the result appears to be incorrect:
```
  Expression: ≈(courant(diffusive_courant, dg, model, Q, Δt, HorizontalDirection()), d_h, rtol=0.0001)
   Evaluated: 5.7754668651338336e-8 ≈ 2.4646041917949998e-5 (rtol=0.0001)
```
cc: @slifer50 @skandalaCLIMA

File: test/DGmethods/courant.jl [hunk 2]
```diff
@@ -131,27 +131,10 @@ let
...
+                @test courant(nondiffusive_courant, dg, model, Q, Δt, VerticalDirection())   ≈ c_v rtol=1e-4
+                @test courant(diffusive_courant,    dg, model, Q, Δt, VerticalDirection())   ≈ d_v rtol=1e-4
```
Comment (simonbyrne) [inline test/DGmethods/courant.jl]: And for this one I get
```
  Expression: ≈(courant(diffusive_courant, dg, model, Q, Δt, VerticalDirection()), d_v, rtol=0.0001)
   Evaluated: 9.24074698421411e-9 ≈ 0.0 (rtol=0.0001)
```

File: test/DGmethods/courant.jl (hunk 1 above)
Comment (simonbyrne) [reply test/DGmethods/courant.jl]: How were `diff_speed_h` and `diff_speed_v` determined?

Comment (slifer50) [reply test/DGmethods/courant.jl]: These were calculated based on the initial conditions for the problem.

**Pull request merged**